docker/run.sh

```diff
@@ -5,9 +5,20 @@ if [[ -z "$WAVEFRONT_URL" ]]; then
   exit 0
 fi
 
-if [[ -z "$WAVEFRONT_TOKEN" ]]; then
-  echo "WAVEFRONT_TOKEN environment variable not configured - aborting startup " >&2
-  exit 0
+authType=""
+if [[ -n "$CSP_APP_ID" && -n "$CSP_APP_SECRET" && -n "$CSP_ORG_ID" ]]; then
+  authType="--cspAppId $CSP_APP_ID --cspAppSecret $CSP_APP_SECRET --cspOrgId $CSP_ORG_ID"
```

Comment:
CSP_ORG_ID is now optional

Reply:
updated

Reply:
Good catch! Updated that, the readme and comments